index.src.html

```diff
@@ -691,16 +691,31 @@ <h3 id="user-mediation">
     Explicit user mediation
   </h3>
 
-  If a feature has privacy or security impacts that are endemic to the feature
-  itself, then one valid strategy for exposing it to the web is to require user
-  mediation before granting an origin access. For instance, [[GEOLOCATION-API]]
-  reveals a user's location, and wouldn't be particularly useful if it didn't;
+  If the security or privacy risk of a feature cannot otherwise be mitigated in
+  a specification, optionally allowing an implementer to prompt a user may
+  provide the best security/privacy possible.  If the specification does not
```

Comment:
I'm not sure if I feel comfortable with saying that this 'may provide the best ... possible'. For example consider the AOM model where this would still result in risks.

Reply:
Revised with https://github.com/w3cping/security-questionnaire/commit/b81de7d4a3731774829e5b619686b313e4538832 to make clear that it is an option and it reduced but does not remove risk.